addon/lib/getScreenshots.js

```diff
@@ -22,7 +22,7 @@ module.exports = Task.async(function*getScreenshots(sites, condition) {
   for (let site of sites) {
     // Don't fetch screenshots if the site doesn't meet the conditions
     if (condition && !condition(site)) {
-      break;
+      continue;
     }
```

Comment:
yep, this actually needs to change so that the screenshots get done in parallel. @k88hudson has a patch that is almost working but it doesn't block the experiment. 👍